src/modules/users/users.controller.ts

```diff
@@ -16,7 +16,11 @@ import { FileInterceptor } from '@nestjs/platform-express';
 import { UsersService } from './users.service';
 import { GetUser, PublicRoute } from '~/common/decorators';
 import { User } from '~/entities';
-import { UserProfileUpdateDto, UserCardDto } from './dto/index';
+import {
+  UserProfileUpdateDto,
+  UserCardDto,
+  UpdateUserAndProfileDto,
+} from './dto/index';
 import { UploadService } from '../upload/upload.service';
 
 @ApiTags('Users')
```

Comment:
I assume you need protect controller, no?

Reply:
All our controllers are protected by default